Mathlib/Data/Multiset/Order.lean

```diff
@@ -0,0 +1,513 @@
+/-
+Copyright (c) 2024 Haitian Wang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Haitian Wang, Malvin Gattinger
+-/
+import Mathlib.Tactic.Linarith
+import Mathlib.Data.Multiset.Basic
+import Mathlib.Logic.Relation
+
+/-!
+# Dershowitz-Manna ordering
```

Comment:
I think "Dershowitz–Manna" should be in the filename, and possibly also in the name of the inductive.